playing-card-detector/newProject/python/cvDev.py

```diff
@@ -0,0 +1,85 @@
+# This file is for developing the cv functions offline, where there is no server to worry about.
+
+import cv2
+import numpy as np
+
+HEIGHT = 540
+WIDTH = 960
+RECT_HEIGHT = 90
+RECT_WIDTH = 120
+
+CARD_AREA_MIN = 5500
+CARD_AREA_MAX = 6000
+SUIT_AREA_MIN = 50
+SUIT_AREA_MAX = 500
+
+def main():
+    path = "playing-card-detector/newProject/sur40screenshots/correctedImage.png"
+    image = cv2.imread(path, cv2.IMREAD_GRAYSCALE)
+    print(analyzeImage(image))
+
+# Takes a full-size image of the SUR40 screen and returns the string that represents the positioning of the cards.
+# The returned string is formatted as follows: "{player}:{position}:{suit}:{rank},{player}:{position}:{suit}:{rank},..."
+# Where player is 1 for the left-side player and 2 is for the right-side player,
+# position is a number 1-6 where 1 is the position at the top of the screen and 6 is the position at the bottom,
+# suit is S for spades, C for clubs, D for diamonds and H for hearts,
+# rank is a number 1-13 where 1 is an ace and 13 is a king.
+def analyzeImage(image):
+    subImages = splitImage(image)
+
+    resultString = ""
+    for i, subImage in enumerate(subImages):
+        player = 1
+        if i > 5: player = 2
+        position = (i % 6) + 1
+        resultString += analyzeSubImage(subImage, player, position)
+
+    #cv2.imshow("image", image)
+    cv2.waitKey(0)
+
+    return resultString
+
+# Splits a full-size image into 12 separate sub-images, one for each position.
+def splitImage(image):
+    subImages = []
+
+    for i in range(12):
+        xOffset = 0
+        if i > 5: xOffset = WIDTH - RECT_WIDTH
```

Comment:
Seems to me like x1 and x2 will be the same values for each side. Am I wrong?
i.e when i <= 5, then xOffset would be just 0 and RECT_WIDTH for all subimages?

Wouldn't we have to do something like:
x1 = xOffset + RECT_WIDTH * i
x2 = xOffset + RECT_WIDTH * (i + 1)
plus some extra compensation for when i > 5?